src/repositories/powerSnapshotRepository.ts

```diff
@@ -0,0 +1,37 @@
+import { PowerSnapshot } from '../entities/powerSnapshot';
+import { Project } from '../entities/project';
+
+export const findInCompletePowerSnapShots = async (): Promise<
+  PowerSnapshot[]
+> => {
+  return PowerSnapshot.createQueryBuilder()
+    .where('"blockNumber" IS NULL')
+    .getMany();
+};
+
+export const findPowerSnapshotById = async (
```

Comment:
Why we need this function and consequently its tests?

Reply:
I needed this query for using in test cases, so I wrote that  once in repository and reuse it in tests